patterns/ecs-v1/rails

```diff
@@ -1,13 +1,13 @@
 RUUID \h{32}
 # rails controller with action
-RCONTROLLER (?<controller>[^#]+)#(?<action>\w+)
+RCONTROLLER (?<[rails][controller][class]>[^#]+)#(?<[rails][controller][action]>\w+)
```

Comment:
I know this is just a carry-forward, but intermixing of literal regex captures and grok-dialect captures makes me twitchy.